### DISCUSSION TITLE
Scroll through copied items?

### DISCUSSION THREAD

**Discussion by sals9** [hluk/CopyQ#1579] [Ideas]
Being able to **scroll** through the items we copied will be a much more efficient way than search only or having a few items in history to be displayed right away... Because there are many times we _forget_ what we copied a few days back, so the new items will not be help and search will not be the best choice. I believe that **scrolling** is the best way to solve this issue. Think Ditto in Windows, they did a great job.

**Comment by hluk** [comment]
I don't understand the request. It should be possible to scroll through items in CopyQ. Do you have an example of the feature?

**Comment by sals9** [comment]
Please see this screenshot >> http://i.imgur.com/59A9QyQ.png
We really need to scroll in this window to speed up our workflow.

Thanks a lot!
Salah

**Comment by sals9** [comment]
Like Ditto in Windows, they made it so efficient >>
http://i.imgur.com/bq2NzKA.png

**Comment by hluk** [comment]
> Please see this screenshot >> http://i.imgur.com/59A9QyQ.png

That's just a menu. Adding a scroll bar there probably won't happen - style of the menu UI widget mainly depends on current system theme.

The menu is only meant to be used to quick access few most recent items. The main application window is meant for browsing scrolling items. Any reason not to use the main window?

**Reply by sals9** [reply]
I'm not using the main window because I need quick access to the copied items with a shortcut without having to click on something or close a window after copying the item (silent mode).

**Reply by hluk** [reply]
It is possible to use the main window this way - i.e. open on shortcut, close when unfocused.

**Reply by sals9** [reply]
But the icon stuck in the bottom bar this way even when it's closed when unfocused. Is there a way to not see it when unfocused?

http://i.imgur.com/ATOKiLD.png

**Reply by hluk** [reply]
Can you also enable "Hide main window" in Layout configuration tab?

**Reply by sals9** [reply]
Now, it works great :)

Thanks a lot!
